src/Microsoft.DotNet.TestFramework/project.json

```diff
@@ -14,7 +14,7 @@
     "NETStandard.Library": "1.5.0-rc2-23922"
   },
   "frameworks": {
-    "netstandard1.3": {
+    "netstandard1.5": {
       "imports": "dnxcore50"
```

Comment:
Now that ASP.NET and CoreFX have both transitioned to `netstandard` I think we could probably remove that import.